package.json

```diff
@@ -12,7 +12,8 @@
     "lint": "yarn lint-server && yarn lint-client",
     "lint-server": "cd server && yarn lint",
     "lint-client": "cd client && yarn lint",
-    "update": "bin/update"
+    "update": "bin/update",
+    "test-server": "cd server && yarn jasmine"
```

Comment:
Can we do `test` everywhere for consistency?

What I mean:
`"test-server": "cd server && yarn test"`

and in server/packange.json:

`"test": "yarn jasmine"`

Thoughts?